test/TestProjects/TypeSchemaMapping/SomeFolder/Generated/Models/CustomizedModel.cs

```diff
@@ -5,60 +5,24 @@
 
 #nullable disable
 
-using System;
-using System.Collections.Generic;
 using NamespaceForEnums;
 
 namespace CustomNamespace
 {
     /// <summary> The Model. </summary>
-    internal partial class CustomizedModel
+    internal partial class CustomizedModel : BaseClassForCustomizedModel
```

Comment:
we now have this change, because we have this base type in the customization code (existed before, not added in this update), but previously we cannot read that from the customization code, the code is generated as if this model does not have a base type.
Now we could recognize types like this `BaseClassForCustomizedModel` which does not correspond to a schema, or a framework type